index.liquid

```diff
@@ -52,18 +52,26 @@
                         <dd><p>{{group.description}}</p><dd>
                     {% endfor %}
                 </dl>
+                <dl class="general-public">
+                <dt><h3>General Public Opening Times</h3><dt>
+                    {% for opening in supermarket.generalHours %}
+                        <dt><strong><p>{{opening.day}}</p></strong><dt>
+                        <dd><p>{{opening.times}}</p><dd>
+                    {% endfor %}
+                </dl>
                 {% if supermarket.notes != "" %}
                     <h4 class="small">Notes</h4>
                     <p class="notes small">{{supermarket.notes}}</p>
                 {% endif %}
                 <p class="small notes" style="margin-top: 0.5em;">View 
-                    {% if supermarket.detailsLink != "" %}<a class="notes" href="{{supermarket.detailsLink}}">Website</a> or{% endif %} 
-                    <a class="notes" href="{{supermarket.twitterLink}}">Twitter</a>
+                    {% if supermarket.detailsLink != "" %}<a class="notes" target="_blank" href="{{supermarket.detailsLink}}">Website</a> or{% endif %} 
+                    <a class="notes" target="_blank" href="{{supermarket.twitterLink}}">Twitter</a>
```

Comment:
Yeah, `target="_blank"` can be a bit of a dodgy A11y and UX issue as it takes away the expected behaviour of clicking a link and can be confusing to users, going to revert that bit 👍 

```suggestion
                    {% if supermarket.detailsLink != "" %}<a class="notes" href="{{supermarket.detailsLink}}">Website</a> or{% endif %} 
                    <a class="notes" href="{{supermarket.twitterLink}}">Twitter</a>
```